src/libraries/System.Reflection/tests/InvokeEmit/System.Reflection.InvokeEmit.Tests.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <TestRuntime>true</TestRuntime>
+    <IncludeRemoteExecutor>true</IncludeRemoteExecutor>
+    <TargetFrameworks>$(NetCoreAppCurrent)-windows</TargetFrameworks>
```

Comment:
Only for windows?

Reply:
How are Mono and Windows related here?

Reply:
I pushed a new commit to add linux.

The mapping was not correct in all cases, especially since the Mono team is working on getting tests to run on Windows.

The new test projects have the same targets as their corresponding original test project except that Browser was left off the SystemReflection tests.